src/app.py

```diff
@@ -5,7 +5,7 @@
 from response import Response
 
 # Function API REST POST create new comment database
-@app.route('/comment_POST', methods=['POST'])
+@app.route('/api/v1/comment', methods=['POST'])
```

Comment:
please use the plural of comment `/api/v1/comments`

Reply:
ok. done